include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -172,6 +172,15 @@ class CMAES
   double& StepSize()
   { return stepSize; }
 
+  //! Get the total number of function evaluations.
+  size_t FunctionEvaluations() const 
+  { return functionEvaluations; }
```

Comment:
```suggestion
  size_t FunctionEvaluations() const  { return functionEvaluations; }
```
This will fit all on one line.